_includes/1.0/api/json/quote.json

```diff
@@ -0,0 +1,64 @@
+{
+  "status": "open",
+  "customer_id": 10000,
+  "billing_contact_id": null,
+  "billing_address": {
+    "street_line1": "1 Rue de Rivoli",
+    "street_line2": "",
+    "street_line3": "",
+    "postal_code": "75004",
+    "city": "Paris",
+    "country": "France",
+    "country_iso_code": "FR",
+    "locality": "",
+    "short_locality": "",
+    "full_address": "1 Rue de Rivoli 75004 Paris France"
+  },
+  "po_number": null,
+  "tax_id": null,
+  "reduce_rate": 3.0,
+  "order_lines": [
+    {
+      "id": 10000,
+      "item_id": 10000,
+      "description": "A service",
+      "quantity": 1.0,
+      "unit_price_wt": 1000.0,
+      "discount_rate": 0.0,
+      "tax_id": 1,
+      "is_taxable": false,
+      "total": 1000.0
+    }
+  ],
+  "public_note": "",
+  "shipping_date": null,
```

Comment:
@fChalmeau Quote have a shipping date?

Reply:
They don't, but the field is into the class the document extends. We could move it out from the abstract class and put it only where it's needed.